docs/_config.yml

```diff
@@ -0,0 +1,39 @@
+# Book settings
+# Learn more at https://jupyterbook.org/customize/config.html
+
+title: napari
+author: The napari team
+logo: images/logo.png
+# only_build_toc_files: true
+
+# Force re-execution of notebooks on each build.
+# See https://jupyterbook.org/content/execute.html
+execute:
+  execute_notebooks: force
+
+# Information about where the book exists on the web
+repository:
+  url: https://github.com/executablebooks/jupyter-book # Online location of your book
+  path_to_book: docs # Optional path to your book, relative to the repository root
+  branch: master # Which branch of the repository should be used when creating links (optional)
+
+# Add GitHub buttons to your book
+# See https://jupyterbook.org/customize/config.html#add-a-link-to-your-repository
+html:
+  use_issues_button: true
+  use_repository_button: true
+
+sphinx:
+  extra_extensions:
+    - sphinx.ext.viewcode
+    - sphinx.ext.napoleon
+    - sphinx.ext.autodoc
+    - sphinx_autodoc_typehints
+    - sphinx.ext.autosummary
+
+  config:
+    autosummary_generate: True
+    autosummary_imported_members: True
+    html_theme: furo
+    templates_path:
+        - '_templates'
```

Comment:
might be nice to override the footer template and give a shout-out to jupyter-book.  (where it says `Built with Sphinx and @pradyunsg's Furo theme.` ... maybe have it say `Built with Sphinx , Jupyter Book and the Furo theme.` with the appropriate links?)